src/shared/docs/en/guides/quickstart.md

```diff
@@ -17,27 +17,27 @@ While this guide assumes general familiarity with such things as text editors, t
 ### tl;dr, you'll need:
 - A (free) GitHub account
 - Any computer running macOS, Windows, or Linux, with:
-  - Node.js 12
-  - npm 6.x
+  - Node.js 12, or later
```

Comment:
I believe Node 14 is required for Sandbox now. Lambdas still support Node 12, but running Sandbox needs Node 14+

Reply:
Yes, good clarification. I will make that change.